app/components/govuk_component/header_component.html.erb

```diff
@@ -1,50 +1,45 @@
 <%= tag.header(class: classes, role: 'banner', data: { module: 'govuk-header' }, **html_attributes) do %>
   <div class="govuk-header__container govuk-width-container">
```

Comment:
The design system component allows setting `containerClasses`; be good if `govuk-components` had a `container_classes` param, the value of which would be appended to the classes provided to this element.

```suggestion
  <div class="govuk-header__container govuk-width-container">
```

Reply:
✔️ 0ffbaf0